src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -612,20 +629,22 @@ public function partialRefundAction(int $orderId, Request $request)
      * @return RedirectResponse
      */
     #[AdminSecurity("is_granted('update', request.get('_legacy_controller')) && is_granted('delete', request.get('_legacy_controller'))")]
-    public function standardRefundAction(int $orderId, Request $request)
-    {
-        $formBuilder = $this->get('prestashop.core.form.identifiable_object.builder.cancel_product_form_builder');
-        $formHandler = $this->get('prestashop.core.form.identifiable_object.standard_refund_form_handler');
+    public function standardRefundAction(
+        int $orderId,
+        Request $request,
+        #[Autowire(service: 'prestashop.core.form.identifiable_object.builder.cancel_product_form_builder')] FormBuilder $formBuilder,
```

Comment:
Can you use the `FormHandlerInterface` for all the injected form handlers instead?